community/omc.html

```diff
@@ -12,14 +12,13 @@
 	<header><h2>Team</h2></header>
 	<div class="entry-content">
 	  <p>
-	  The OpenSSL development team consists of people who have commit
-	  access to the source repository.
-	  Policies and procedures are adopted after a majority vote of
-	  the dev-team.
+          The
+          <a href="/policies/bylaws.html#OMC">OpenSSL Management Committee</a>
+          represents the official voice of the project. All official
+          OMC decisions are taken on the basis of a vote.</p>
 
 	  <p>
-	  The current OpenSSL development team consists of (in alphabetical
-	  order):
+	  The current OMC consists of (in alphabetical order):
```

Comment:
Don't we need a separate list of committers too? At the moment the list will be largely the same, but very soon it will be different.